entrypoint.sh

```diff
@@ -22,4 +22,7 @@ rm -rf Makefile
 rm -rf .github
 rm -f cloudbuild.yaml
 rm -f .travis.yml
+rm -f crowdin.yml
+rm -rf docs
```

Comment:
Not sure about the docs folder, isn't it supposed to be used to store user documentation when we read the devdocs.